doc/api-docs/conf.py

```diff
@@ -77,7 +69,7 @@
 #
 # This is also used if you do content translation via gettext catalogs.
 # Usually you set "language" from the command line for these cases.
-language = None
+language = 'en'
```

Comment:
This was causing an error in the ReadTheDocs build, probably due to a sphinx update.